app/src/main/res/layout/profile_view.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/profileView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/UC_red"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/profile_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:background="@color/black"
+        android:gravity="center"
+        android:padding="20dp"
+        android:text="@string/profile"
+        android:textAppearance="@style/TextAppearance.AppCompat.Large"
+        android:textColor="@color/whiteTextOnBlackBackground"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/avatarPhotoView"
+        android:layout_width="match_parent"
+        android:layout_height="168dp"
+        android:contentDescription="@string/profile_photo"
+        tools:srcCompat="@tools:sample/avatars" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/btnCamera"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="end"
+        android:layout_marginStart="0dp"
+        android:layout_marginTop="0dp"
+        android:layout_marginEnd="175dp"
+        android:backgroundTint="@color/black"
+        android:src="@android:drawable/ic_menu_camera"
+        android:tint="@color/whiteTextOnBlackBackground"
+        app:borderWidth="0dp"
+        app:elevation="4dp"
+        app:fabSize="normal" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="75dp"
+        android:layout_marginTop="30dp"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/txtPersonName"
+            android:layout_width="304dp"
+            android:layout_height="63dp"
+            android:autofillHints="Enter your profile name here..."
```

Comment:
This should be factored to `@strings`

Reply:
This is factored in strings.xml. It shows up in gray text and when I click on it, it shows that it is already factored.